src/core/bignum.cc

```diff
@@ -58,6 +58,24 @@ CL_DEFUN Bignum_sp Bignum_O::make(const string &value_in_string) {
   return ((bn));
 };
 
+string Bignum_O::valueAsString() const {
+  stringstream      ss;
+  int               ibase = 10;
+  char            * buffer = NULL;
+
+  buffer = mpz_get_str( NULL, ibase, this->_value.get_mpz_t() );
```

Comment:
Does this leak memory?   Perhaps we should have a predefined buffer to mpz_get_str into